### PR TITLE
VIDEO-11360 LocalMediaTrack.isMuted and Room.refreshInactiveMedia.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

This PR implements the following [proposal](https://docs.google.com/document/d/1hVDcsMLAAEL2uuES5Dzl6KXRtnqMDt-5hxZBd8u9kW0/edit?usp=sharing) for working around [VIDEO-11360](https://issues.corp.twilio.com/browse/VIDEO-11360).

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [ ] Ready for review

### CODE REVIEW

File: lib/media/track/localaudiotrack.js [hunk 1]
```diff
@@ -183,7 +189,7 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
   }
 
   /**
-   * Disable the {@link LocalAudioTrack}. This is effectively "mute".
```
Comment (PikaJoyce) [inline lib/media/track/localaudiotrack.js:186]: I think if we remove `This is effectively "mute"`, perhaps we need to be more clear about what the differences are between mute/unmute and disable/enable are.

Perhaps adding 
```This is effectively "pause"```

File: lib/media/track/localaudiotrack.js [hunk 2]
```diff
@@ -192,13 +198,12 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
   }
 
   /**
-   * Enable the {@link LocalAudioTrack}. This is effectively "unmute".
```
Comment (PikaJoyce) [inline lib/media/track/localaudiotrack.js:195]: Same as above comment.

Perhaps adding
`This is effectively "play"`

File: lib/media/track/localaudiotrack.js [hunk 3]
```diff
@@ -264,18 +269,25 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
 }
 
 /**
- * The {@link LocalAudioTrack} was disabled, i.e. "muted".
```
Comment (PikaJoyce) [inline lib/media/track/localaudiotrack.js:267]: Same as above comments about renaming "muted" to "paused" and "unmuted" to "play".

File: lib/media/track/localaudiotrack.js [hunk 4]
```diff
@@ -264,18 +269,25 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
...
+ * The {@link LocalAudioTrack} was disabled.
  * @param {LocalAudioTrack} track - The {@link LocalAudioTrack} that was
  *   disabled
  * @event LocalAudioTrack#disabled
  */
 
 /**
- * The {@link LocalAudioTrack} was enabled, i.e. "unmuted".
```
Comment (PikaJoyce) [inline lib/media/track/localaudiotrack.js:274]: Same as above comment

**Review by PikaJoyce — CHANGES_REQUESTED** [review]
Awesome work Manjesh!!! Left a couple of comments/questions mostly. Otherwise, implementation looks great!

File: lib/media/track/localaudiotrack.js [hunk 5]
```diff
@@ -14,15 +14,21 @@ const LocalMediaAudioTrack = mixinLocalMediaTrack(AudioTrack);
  * {@link LocalAudioTrack#stop}.
  * @extends AudioTrack
  * @property {Track.ID} id - The {@link LocalAudioTrack}'s ID
+ * @property {boolean} isMuted - Whether or not the audio source has stopped sending samples to the
+ *   {@link LocalAudioTrack}; This can happen when the microphone is taken over by another application,
+ *   mainly on mobile devices; When this property toggles, then <code>muted</code> and <code>unmuted</code>
+ *   events are fired appropriately
```
[the remark this answers is not in the document]
Comment (manjeshbhargav) [reply lib/media/track/localaudiotrack.js:20]: Typically we don't use periods for describing properties or methods.

**Review by PikaJoyce — APPROVED** [review]
LGTM!

**Pull request merged**